_source/user-guide/infrastructure-monitoring/rollups.md

```diff
@@ -9,34 +9,87 @@ contributors:
+###### On this page
+{:.no_toc}
 
-Take CPU usage for example, you probably don't need the exact CPU usage at
-10 second intervals after a week has passed.
-But you would want to see CPU usage steadily creep up at a 1.5% increase per hour over a week.
+1. toc list
+{:toc}
+
+
+### Rollups explained
+
+Data rollups offer an effective process for compressing metrics without losing the original extremes. Rollups always keep the original max, min, and average values of your metrics so you can graph the data more accurately despite its compression.
+
+When compacting the metrics, Logz.io calculates 4 indicators for each unique data point (UDP):
+
+* Average
+* Max
+* Min
+* Sum
+
+When graphing a data point, you can select either the raw value or the rollup average, max, min, or sum. For example, here are the options for graphing the number of CPU cores per request.
+
+![Select Grafana data point as average/max/min/sum in Logz.io](https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana/rollups-dp.png)
+
+The method of calculating the rollups is explained in the next section.
 
-| Time since data was first-indexed          | Maximum granularity|
+### Data granularity over time
+
+When you're investigating incidents, the granularity of the data will vary depending on how much time has passed, as shown in the table below.
+
+| Time since data was first indexed          | Maximum granularity|
 |-----------------------|--------------------------------------------|
 | Up to 72 hours (3 days)| Raw data (1 second interval between data points, at minimum)|
 | 3 - 7 days            | 1 minute rollups             |
 | 7 - 30 days           | 10 minute rollups            |
 | 30 day or more        | 1 hour rollups               |
 
-When you're investigating incidents, the granularity of the data will vary depending on how much time has passed. As long as you're within 3 days of the incident, you can drill down on the raw data points. As you move past the 3 day mark, you'll be able to drill down on the data at a granularity of 1 minute intervals. Once a week has passed, you'll be able to drill down to a maximum resolution of 10 minute intervals. Once a month or more has passed, the data is at its maximum compaction, and you can drill down on the hourly metrics. 
+As long as you're within 3 days of the incident, you can drill down on the raw data points. As you move past the 3 day mark, you'll be able to drill down on the data at a granularity of 1 minute intervals. Once a week has passed, you'll be able to drill down to a maximum resolution of 10 minute intervals. Once a month or more has passed, the data is at its maximum compaction, and you can drill down on the hourly metrics.
 
-### Available data points
+### Default rollups
+
+The following is a list of rollup configs that are pre-configured and available by default:
+
+* Aws (Major services and namespaces, including EC2, S3, RDS, SQS)
```

Comment:
We'll probably need to specify exactly which servers are available both on AWS and Azure